src/rust-2018/macros/macro-changes.md

```diff
@@ -78,3 +80,158 @@ struct Bar;
 
 This only works for macros defined in external crates.
 For macros defined locally, `#[macro_use] mod foo;` is still required, as it was in Rust 2015.
+
+### Local helper macros
+
+Sometimes it is helpful or necessary to have helper macros inside your module. This can make
+supporting both versions of rust more complicated.
+
+For example, let's make a simplified (and slightly contrived) version of the `log` crate in 2015
+edition style:
+
+```rust,ignore
+pub struct LogLevel {
+    Warn,
+    Error
+}
+
+#[doc(hidden)]
+#[macro_export]
+macro_rules! log {
```

Comment:
I would rename this macro to `private_log!` or `log_impl!` or `__log!` to make it clear that importing it is not a reasonable fix (because the real log crate does export a public `log!` macro).